src/featurizations/weavefeaturization.jl

```diff
@@ -2,7 +2,7 @@
 
 struct WeaveFeaturization <: AbstractFeaturization
     atom_features::Vector{AtomFeature}
-    pair_features::Vector{PairFeature}
+    pair_features::Vector{GeneralPairFeature}
```

Comment:
probably `Vector{<:AbstractPairFeature}`

Reply:
Fixed in 7e53044.